### DISCUSSION TITLE
Suggestion for Ctrl+Shift+X and icon color

### DISCUSSION THREAD

**Discussion by akoppa** [hluk/CopyQ#1858] [Ideas]
According to the FAQ, to disable storing clipboard, to temporarily disable storing the clipboard we use Ctrl+Shift+X shortcut. To re-enable the functionality we use the same shortcut.

That's fine, but it would be great that this command will also change the CopyQ icon color, to something like red or grey, as a reminder that the CopyQ function is disabled.

**Comment by hluk** [comment]
> it would be great that this command will also change the CopyQ icon color

See #1800.

**Comment by akoppa** [comment]
Thank you! 
At first it didn't work for me, but after a second try it worked just fine. Really it was as easy as 1,2,3.
